external/storm-jdbc/src/main/java/org/apache/storm/jdbc/common/HikariCPConnectionProvider.java

```diff
@@ -40,6 +44,10 @@ public synchronized void prepare() {
             Properties properties = new Properties();
             properties.putAll(configMap);
             HikariConfig config = new HikariConfig(properties);
+            if(properties.containsKey("dataSource.url"))
+                LOG.info("DataSource Url: " + properties.getProperty("dataSource.url"));
+            else if (config.getJdbcUrl() != null)
+                LOG.info("JDBC Url: " + config.getJdbcUrl());
```

Comment:
Shouldn't this printout be independent of the presence of dataSource.url?
i.e. just an {{if}} instead of {{else if}}